Author: Joachim Reichel <reichel@debian.org>
Description: use move instead of rename to avoid EXDEV, see #336808 and #337396
--- a/src/normalize-mp3.in
+++ b/src/normalize-mp3.in
@@ -47,6 +47,7 @@
 #######################################################################
 
 use Fcntl;
+use File::Copy;
 
 sub usage {
     print <<EOF
@@ -214,7 +215,7 @@
 	    close(IN);
 	    close(OUT);
 	    unlink $fname;
-	    rename($tmpfile, $fname)
+	    move($tmpfile, $fname)
 		|| die "Can't rename temp file, leaving in $tmpfile, stopped";
 	}
     }
